floodlight/io/skillcorner.py

```diff
@@ -0,0 +1,264 @@
+import json
+from typing import Dict, Tuple, Union, List
+
+import numpy as np
+import pandas as pd
+
+from floodlight import Teamsheet, Code, XY, Pitch
+
+
+def _get_meta_data(
+    match_data: Dict[str, Union[List, str, Dict, int]]
+) -> Tuple[int, int, List[int], int]:
+    """
+    Extract metadata from match data including team IDs, referee IDs, and ball ID.
+
+    Parameters
+    ----------
+    match_data : dict
+        A dictionary containing match data with keys 'home_team', 'away_team',
+        'referees', and 'ball'.
+
+    Returns
+    -------
+    home_team_id : int
+        The ID of the home team.
+    away_team_id : int
+        The ID of the away team.
+    referee_ids : list of int
+        A list of trackable object IDs for the referees.
+    ball_id : int
+        The trackable object ID for the ball.
+
+    """
+    home_team_id = match_data["home_team"]["id"]
+    away_team_id = match_data["away_team"]["id"]
+
+    referee_ids = [x["trackable_object"] for x in match_data["referees"]]
+    ball_id = match_data["ball"]["trackable_object"]
+
+    return home_team_id, away_team_id, referee_ids, ball_id
+
+
+def get_team_sheets(
```

Comment:
one last comment, same as with the other functions that are now private:
- for all other io module functions we use "teamsheets" instead of "team_sheets" in the function name
- if public, it's still not quite clear why one is called meta data, and the other match data, and how that match data is supposed to look like. maybe as the easiest fix, just add to the docstring parameter explanation that ``match_data`` is "A dictionary [...] as returned by the ``_get_meta_data`` function.

Reply:
I think at this point this function should just be private as well. I still added a bit of explanation to each of the functions dealing with "match_data".